tdrs-frontend/src/components/Header/Header.jsx

```diff
@@ -53,33 +53,41 @@ function HeaderComp() {
                 </a>
               </li>
             </ul>
-            {user && user.email && (
-              <div className="usa-nav__secondary">
-                <ul className="usa-nav__secondary-links">
-                  <li className="usa-nav__secondary-item">
+            <div className="usa-nav__secondary">
+              <ul className="usa-nav__secondary-links">
+                <li
+                  className={`${
+                    user && user.email ? 'display-block' : 'display-none'
+                  } usa-nav__secondary-item`}
+                >
+                  {user && user.email && (
                     <a href="/">
                       <FontAwesomeIcon
                         className="margin-right-1"
                         icon={faUserCircle}
                       />
-                      {user.email}
-                    </a>
-                  </li>
-                  <li className="usa-nav__secondary-item">
-                    <a
-                      className="sign-out-link"
-                      href={`${process.env.REACT_APP_BACKEND_URL}/logout/oidc`}
-                    >
-                      <FontAwesomeIcon
-                        className="margin-right-1"
-                        icon={faSignOutAlt}
-                      />
-                      Sign Out
+                      {user && user.email}
```

Comment:
This is so that keyboard navigation works for navigation of the menus. With the way that the USWDS JS is written, it cannot dynamically render the secondary links and have them available for tabbing. At least one has to be there so that everything can be tabbed through. The profile `li` will be hidden when the user is not logged in.